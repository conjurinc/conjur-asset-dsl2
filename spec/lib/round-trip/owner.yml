- !owner
  record: !variable foobar
  role: !group developers

- !owner
  record: !variable 
    id: baz
  role: !group
    id: developers
