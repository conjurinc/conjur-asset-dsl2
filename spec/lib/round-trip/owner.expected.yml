---
- !owner
  record: !variable
    id: foobar
  role: !group
    id: developers
- !owner
  record: !variable
    id: baz
  role: !group
    id: developers
